include/snowflake/client.h

```diff
@@ -64,17 +64,19 @@ extern "C" {
 /**
  * Login timeout in seconds
  */
+// make the login timetout defaults to 300 to be inline with retry timeout
+// while customer can reduce it as needed
 #define SF_LOGIN_TIMEOUT 300
 
  /**
- * network timeout other than login requests
+ * retry timeout in seconds
  */
-#define SF_NETWORK_TIMEOUT 120
```

Comment:
Why you removed this?

Reply:
roll back the change made in previous PR to keep the behavior of existing connection parameters.